### ISSUE TITLE
Training data of model-based filtering

### ISSUE THREAD

**Issue by Yu-Shi** [mlfoundations/dclm#74]
Hi authors, I'm interested in researching the model-based filtering process and reproducing the training of the fasttext model in the DCLM-baseline. Could you please provide the training data for this?

**Comment by Yu-Shi** [comment]
Or, Can you provide more information on the construction of the training data, so that others can reproduce a similar model?

**Comment by Yu-Shi** [comment]
Hi authors, are there any updates on this? thank you

**Comment by afang-story** [comment]
I don't think we can release the data, but we will update if this changes. You can find OpenHermes-2.5 on Hugging Face, and instructions for reproducing the ELI5 portion are in Appendix I.1 in the paper.
